frontend/src/scenes/funnels/Funnel.scss

```diff
@@ -0,0 +1,85 @@
+@import '~/vars';
+
+.funnels-empty-state {
```

Comment:
This file extracts verbatim the relevant styles that are part of the new components from the legacy `FunnelViz.scss`